### ISSUE TITLE
[Mesh] mesh element generation for unstructured meshes

### ISSUE THREAD

**Issue by MuellerSeb** [GeoStat-Framework/GSTools#91]
With the new mesh class from #65, we should think about generating simple mesh elements from the given points. This could be done with:

https://docs.scipy.org/doc/scipy-1.1.0/reference/generated/scipy.spatial.Delaunay.html

This could help visualizing with pyvista, to get a good mesh representation. otherwise I would stick to vertices as mesh elements.

**Comment by LSchueler** [comment]
Yeah that's a good idea and it should be pretty easy to simply wrap the Delaunay class from Scipy.

**Comment by banesullivan** [comment]
PyVista also gives access to VTK's `vtkDelaunay2D`/`vtkDelaunay3D` filters - just an FYI. Probably safer to use scipy as a dependency

**Comment by banesullivan** [comment]
> This could help visualizing with pyvista, to get a good mesh representation. otherwise I would stick to vertices as mesh elements.

But if this is strictly for a visualization, then you might as well just task PyVista with triangulating/tetrahedralizing the meshes and not worry about maintaining a Delaunay algorithm in here.
